tests/testthat/test-caption.R

```diff
@@ -0,0 +1,217 @@
+library(testthat)
+library(dplyr)
+
+context("test-caption")
```

Comment:
`context` is deprecated but may still worth adding for consistency with other tests.

Reply:
Thanks; I've been including it but looks like testthat is using the file name if you don't include it which is what I wanted. But I should update to the new system at some point.